modules/gdscript/editor/gdscript_translation_parser_plugin.cpp

```diff
@@ -52,10 +52,16 @@ Error GDScriptEditorTranslationParserPlugin::parse_file(const String &p_path, Ve
 	ids = r_ids;
 	ids_ctx_plural = r_ids_ctx_plural;
 	Ref<GDScript> gdscript = loaded_res;
+	ERR_FAIL_COND_V_MSG(!gdscript->is_valid(), ERR_FILE_CANT_OPEN, "Failed to load GDScript.");
```

Comment:
Should this not be:
```suggestion
	ERR_FAIL_COND_V_MSG(!gdscript.is_valid(), ERR_FILE_CANT_OPEN, "Failed to load GDScript.");
```
I.e.
```suggestion
	ERR_FAIL_COND_V_MSG(gdscript.is_null(), ERR_FILE_CANT_OPEN, "Failed to load GDScript.");
```
Or:
```suggestion
	ERR_FAIL_COND_V_MSG(gdscript.is_null() || !gdscript->is_valid(), ERR_FILE_CANT_OPEN, "Failed to load GDScript.");
```
Is this check for the script or the resource?